File: Android/app/src/main/java/com/diceroller/DiceRollerLogModel.java
```java
package com.diceroller;

import java.util.ArrayList;
import java.util.List;

public class DiceRollerLogModel {

    private List<String> rolls;

    public DiceRollerLogModel() {
        this.rolls = new ArrayList<>();
    }

    public List<String> getRolls() {
        return rolls;
    }

    public void addRollToLog(String roll) {
        this.rolls.add(roll);
    }
}
```

File: iOS/DiceRoller/DiceRollerLogModel.swift
import Foundation

class DiceRollerLogModel {
    var rolls: [String] = []
    
    func addRollToLog(roll: String) {
        rolls.append(roll)
    }
    
    func getRolls() -> [String] {
        return rolls
    }
    
    func clearLog() {
        rolls.removeAll()
    }
}